consonance-arch/sql/schema.sql

```diff
@@ -1,61 +1,243 @@
 --
--- Name: provision_id_seq; Type: SEQUENCE; Schema: public; Owner: seqware
+-- PostgreSQL database dump
 --
 
-CREATE SEQUENCE provision_id_seq
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+--
+-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: 
+--
+
+CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
+
+
+--
+-- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: 
+--
+
+COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
+
+
+SET search_path = public, pg_catalog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: extra_files; Type: TABLE; Schema: public; Owner: queue_user; Tablespace: 
+--
+
+CREATE TABLE extra_files (
+    job_id integer NOT NULL,
+    content text,
+    path text NOT NULL
+);
+
+
+ALTER TABLE extra_files OWNER TO queue_user;
+
+--
+-- Name: ini_params; Type: TABLE; Schema: public; Owner: queue_user; Tablespace: 
+--
+
+CREATE TABLE ini_params (
+    job_id integer NOT NULL,
+    value text,
+    key text NOT NULL
+);
+
+
+ALTER TABLE ini_params OWNER TO queue_user;
+
+--
+-- Name: job; Type: TABLE; Schema: public; Owner: queue_user; Tablespace: 
+--
+
+CREATE TABLE job (
+    job_id integer NOT NULL,
+    container_image_descriptor text,
+    container_runtime_descriptor text,
+    create_timestamp timestamp without time zone,
+    end_user text,
+    flavour text,
+    ini text,
```

Comment:
Yep, once I get the new web client and worker, the old ini gets removed. 
But until that gets hooked up, I'll just leave the old versions for testing purposes.